official/resnet/resnet_run_loop.py

```diff
@@ -266,8 +277,19 @@ def exclude_batch_norm(name):
     if multi_gpu:
       optimizer = tf.contrib.estimator.TowerOptimizer(optimizer)
 
+    if loss_scale != 1:
+      # When computing fp16 gradients, often intermediate tensor values are
+      # so small, they underflow to 0. To avoid this, we multiply the loss by
+      # loss_scale to make these tensor values loss_scales times bigger.
+      scaled_grad_vars = optimizer.compute_gradients(loss * loss_scale)
+      unscaled_grad_vars = [(grad / loss_scale, var)
```

Comment:
Can you comment to explain the second step here too?

Reply:
Done.